applications/nrf_desktop/src/util/desktop_suit.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include "desktop_suit.h"
+#include <sdfw/sdfw_services/suit_service.h>
+
+int suit_get_installed_manifest_seq_num(unsigned int *seq_num)
```

Comment:
I think that prefix should be `desktop_suit_*` so we will not mistake it with the APIs from the SUIT service.

Reply:
Ignore as in teams discussion it was decided that this util will be removed.